src/Cli/ConfigGenerator.cs

```diff
@@ -32,24 +32,35 @@ public class ConfigGenerator
         /// </summary>
         public static bool TryGenerateConfig(InitOptions options)
         {
-            if (!TryGetConfigFileBasedOnCliPrecedence(options.Config, out string runtimeConfigFile))
+            string runtimeConfigFile = RuntimeConfigPath.DefaultName;
+            string? environmentValue = Environment.GetEnvironmentVariable(RuntimeConfigPath.RUNTIME_ENVIRONMENT_VAR_NAME);
+            if (!string.IsNullOrWhiteSpace(options.Config))
+            {
+                _logger.LogInformation("Generating user provided config file: {configFileName}", options.Config);
```

Comment:
```suggestion
                _logger.LogInformation("Generating user provided config file path: {configFileName}", options.Config);
```

either add path or name.. otherwise it reads as if the user is providing the actual file